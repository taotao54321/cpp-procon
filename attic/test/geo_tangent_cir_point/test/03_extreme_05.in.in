-256 987
0 0 1
