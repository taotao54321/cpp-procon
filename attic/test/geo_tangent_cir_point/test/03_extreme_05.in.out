-0.9682161791 -0.2501148347
0.9677237317 0.2520134502
